src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
+  background-color: #fff;
+  border: 1px solid $light-grey-color;
+  padding: 32px 16px 16px;
+  transition: $hover-duration;
```

Comment:
Ensure that the SCSS variable `$hover-duration` is defined and correctly imported in your SCSS setup to prevent any compilation errors.